### PR TITLE
Bump onecodex and scikit-bio versions

### PR DESCRIPTION
**Pull request by alenajk**
Updating dependencies and base image version (from Python 3.8 to 3.9) to use most recently released `Onecodex` library version (`0.16.0`)

### CODE REVIEW

File: test/notebooks/example_expected.pdf
Comment (petraszd) [inline test/notebooks/example_expected.pdf:1]: `example_expected.pdf` now contains "WARNING" line.

Comment (alenajk) [reply test/notebooks/example_expected.pdf:1]: Updated

**Comment by alenajk** [comment]
@jairideout whoops, those got added erroneously. I updated it / took them out.

File: test/pytest.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[pytest]
```
Comment (jairideout) [inline test/pytest.ini:1]: Is this file needed now that we're filtering the deprecation warnings in the example notebook?

**Review by jairideout — APPROVED** [review]

File: test/pytest.ini (hunk 1 above)
Comment (alenajk) [reply test/pytest.ini]: It looks like not needed - I removed it.
